robocop/checkers/misc.py

```diff
@@ -261,6 +261,33 @@
         
         """,
     ),
+    "0916": Rule(
+        RuleParam(
+            name="max_width",
+            default=80,
+            converter=int,
+            desc="Maximum width of IF below which it will be recommended to use inline IF",
```

Comment:
Width calculated in which unit? Line length?
Then it should be stated here, like `"Maximum width of IF (in characters) below which it will be recommended to use inline IF" or something like this.